index.md

```diff
@@ -4,14 +4,27 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
 
+> ## Looking for Beta Testers!
+> **We are currently looking for volunteers to test this lesson!**
+> If you are a certified Carpentries Instructor and you would like to
```

Comment:
Do we need to require they be certified?

Reply:
It may make things easier if we ever decide to submit the lesson for review in The Carpentries Lab. But I guess at this stage we would appreciate feedback from anyone who wants to teach it! So I will remove that wording from the callout.